### PR TITLE
add form component and connect the form to the activitylist

### PR DESCRIPTION
**Pull request by jonas8900**
add a dumb form component for reusing in the future
connect the form to the activitylist
add local storage state to store the activitycards 
add styles to the inputfield icons, because the whole application will use this icon and inputfield

### CODE REVIEW

File: components/Activitylist.js [hunk 1]
```diff
@@ -1,21 +1,55 @@
...
+    }
+  );
+
+  function handleSubmitActivity(event) {
+    event.preventDefault();
+
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    const inputData = {
```
Comment (F-Kirchhoff) [inline components/Activitylist.js:23]: maybe you can find a more fitting Name than inputData, but for now its fine.

File: components/Activitylist.js [hunk 2]
```diff
@@ -1,21 +1,55 @@
...
+      <main>
+        <StyledHeadline>Friends</StyledHeadline>
+        <StyledActivitySection>
+          <StyledHeadlineForSubpages>Aktivitäten</StyledHeadlineForSubpages>
+          {activityCards.map((activity) => (
+            <StyledList key={activity.id}>
+              <StyledListItem>
+                <StyledListItemHeadline>{activity.name}</StyledListItemHeadline>
+              </StyledListItem>
+            </StyledList>
```
Comment (F-Kirchhoff) [inline components/Activitylist.js:43]: right now you create a ul element for every single li element. But we want all li elements in a single ul element. Please update your code accordingly.

File: components/Form.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import styled from "styled-components";
+
+export default function Form({ name, onSubmit, placeholder, type }) {
+  return (
+    <>
```
Comment (F-Kirchhoff) [inline components/Form.js:7]: you don't need the fragments

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Hey Jonas,
looking almost ready. Please update your code as mentioned.

File: components/Activitylist.js (hunk 2 above)
Comment (jonas8900) [reply components/Activitylist.js:43]: thank you ! :), yes thats a bit confusing! I've done that too complicated, i would like to have multiple list items for each card like a "plan" button, a "like" button etc.

File: components/Activitylist.js (hunk 1 above)
Comment (jonas8900) [reply components/Activitylist.js:23]: for Sure! I'll changed that to "newActiviry"

File: components/Form.js (hunk 3 above)
Comment (jonas8900) [reply components/Form.js:7]: you are right... the first thing i do is to set fragments for no reason, i don't know why. Thanks for your time !

**Review by F-Kirchhoff — APPROVED** [review]
LGTM 🦊

**Pull request merged**